src/Excepts.java

```diff
@@ -0,0 +1,63 @@
+import java.util.Random;
+
+// Custom checked exception
+public class NotEnoughGas extends Exception {
+    public NotEnoughGas(String message) {
+        super(message);
+    }
+}
+
+// Custom unchecked exception
+public class Explosion extends RuntimeException {
+    public Explosion(String message) {
+        super(message);
+    }
+}
+
+class Car {
+    private int fuel;
+    private int kilometers;
+    private Random random;
+
+    public Car() {
+        this.fuel = 35;
+        this.kilometers = 0;
+        this.random = new Random();
+    }
+
+    public void fill() {
+        if (random.nextInt(10) == 0) {
+            throw new Explosion("BOOOOOOM!");
+        }
+
+        int randomFillofFuel = random.nextInt(35 - 15 + 1) + 15;
```

Comment:
1. `s/randomFillofFuel/randomFillOfFuel/`
2. add a comment saying that this should return a random value from 15 to 36